src/Command/LoadDataFixturesDoctrineODMCommand.php

```diff
@@ -50,16 +54,28 @@ protected function configure(): void
 You can also choose to load only fixtures that live in a certain group:
 
     <info>php %command.full_name%</info> <comment>--group=group1</comment>
+
+If the collection uses search indexes or encryption, you can use the <info>--purge-with-delete</info> option to keep the collections instead of dropping them when purging the database:
+
+  <info>php %command.full_name%</info> --purge-with-delete
 EOT
         );
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $dm = $this->getManagerRegistry()->getManager($input->getOption('dm'));
+        assert($dm instanceof DocumentManager);
+
         $ui = new SymfonyStyle($input, $output);
 
-        if ($input->isInteractive() && ! $input->getOption('append')) {
+        if ($input->getOption('purge-with-delete')) {
+            if ($input->getOption('append')) {
+                $ui->error('The --purge-with-delete option cannot be used with the --append option.');
+
+                return self::INVALID;
+            }
+        } elseif ($input->isInteractive() && ! $input->getOption('append')) {
```

Comment:
should this really be a `elseif` ? Using `--purge-with-delete` without `--append` in an interactive session should still ask for confirmation for purging. To me, it should be a `if`

Reply:
I think that if we have the `--purge-with-delete` option, we are explicitly asking to purge, so there is no need to confirm. But for me, we need an other option for `--purge-with-drop` that skip the confirmation. 

That said, it would be better to have a single option, `--purge`, with an optional value: `--purge`, `--purge=drop`, `--purge=delete`. It would skip the confirmation.